README.md

```diff
@@ -44,7 +45,11 @@ School Library
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your description in the readme file more detailed to make your  readme descriptive and professional 😊